### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPFBrush_Contents_ScreenReader: Using narrator the proper role as "treeview item" is not announced for options of "Contents" for screen reader users.

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1454]
Copied from VSO: [947814](https://devdiv.visualstudio.com/DevDiv/_queries/edit/947814/)

Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: .NET Core 3.0 (WPF and ASP.NET Core)

Application version: VS 2019 Preview 4.7.03190

Windows Version: Windows10

ScreenReader Version: Narrator, NVDA 2019.1.1

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.
2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link.

Repro Steps:

1) Open Visual Studio.

2) Hit on File and select Open->Project/Solution.

3) Select the "Brushes" solution and open it.

4) From the Solution Explorer, right click on the file under the solution "Brushes".

5) Select "Debug" and hit "Start new Instance".

6) Main Window for "Brush Examples" solution will appear.

7) Navigate to contents section and Expand it.

8) Start screen reader.

9) Navigate to the each of the options and listen if it's role is announced or not.

Actual:

Using narrator when screen reader users navigate through the options of the "Contents" a proper role as treeview items is not announced to the screen reader users.

Expected:

When the screen reader users navigates to the options of the "Contents" its role as treeview items should be announced.

User Impact:

If proper role of the interactive elements is not announced to the screen reader user then they might not know how to interact with them.

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 4.1.2

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Comment by rladuca** [comment]
This is just a bug in the `ItemContainerStyle` for the TreeView that the application is setting, not a bug in WPF.

Since the source for the data is a bunch of XML nodes, you need to bind AutomationProperties.Name to the Title from the DataContext.  The DataContext being an XmlElement, the binding is:
```
<Setter Property="AutomationProperties.Name" Value="{Binding XPath=@Title}"/>
```

This sets the automation name appropriately for Narrator, etc.
